packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -59,3 +59,22 @@ def test_ip_per_container(cluster):
         app_port = app_definition['container']['docker']['portMappings'][0]['containerPort']
         cmd = '/opt/mesosphere/bin/curl -s -f http://{}:{}/ping'.format(service_points[1].ip, app_port)
         ensure_routable(cmd, service_points)()
+
+
+def test_ip_per_container_with_named_vip(cluster):
```

Comment:
There were three tests prior to this fourth one. Here is my understanding of those three tests:
1. minuteman routes to VIP -- tests reachability to VIP without overlay.
2. minuteman routes to named VIP -- same as 1 but with a named VIP.
3. IP per container -- reachability to a container IP without load balancing.
4. IP per container with named VIP -- reachability to a container IP with load balancing.